hubspot/api.py

```diff
@@ -266,3 +269,189 @@ def make_product_sync_message(product_id):
     product = Product.objects.get(id=product_id)
     properties = ProductSerializer(product).data
     return [make_sync_message(product_id, properties)]
+
+
+def sync_object_property(object_type, property_dict):
+    """
+    Create or update a new object property
+
+
+    Args:
+        object_type (str): The object type of the property (ie "deals")
+        property_dict (dict): The attributes of the property
+
+    Returns:
+        dict:  The new/updated property attributes
+    """
+    required_fields = {"name", "label", "groupName"}
+
+    missing_fields = required_fields.difference(property_dict.keys())
+    if missing_fields:
+        raise KeyError(
+            f"The following property attributes are required: %s",
```

Comment:
Is this supposed to be an f-string or was `format()` supposed to be called on it?

Reply:
I probably started it as an f-string, then changed it to use `%s` and forgot to get rid of the `f`.  Will remove.